Microservices/user-service/src/main/java/app/sportahub/userservice/controller/auth/AuthController.java

```diff
@@ -44,5 +46,14 @@ public LoginResponse loginUser(@RequestBody LoginRequest loginRequest) {
     public TokenResponse refreshToken(@RequestBody RefreshTokenRequest tokenRequest) {
         return authService.refreshToken(tokenRequest);
     }
+
+    @PutMapping("/send-verification-email")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "send verification email",
+            description = "Sends a verification email to the specified user. This email contains a link or token that the user must use to confirm their email address and complete the registration."
+    )
+    public void sendVerificationEmail(@RequestBody Map<String, Object> payload) {
+        authService.sendVerificationEmail(payload.get("email").toString());
+    }
```

Comment:
Can you use a response model under the /dto/response for the request body? Validate using Jakarta please.

Reply:
EDIT: @patrickmac3 I meant to use a request model un the `/dto/request` package